frodo.js

```diff
@@ -4,29 +4,31 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-Good job on updating the documentation along with the changes. It's very thorough and helpful!